### ISSUE TITLE
How to add data loss to neural network?

### ISSUE THREAD

**Issue by CharukaWick** [lululxvi/deepxde#1952]
I have a system of parametric differential equations with observed data. I would like to know how to add data loss to the neural network.

For an example, could you explain how to add data loss to the "Inverse problem for the Lorenz system"? Because the system of differential equations that I'm working with is similar to the Lorenz system.

Any explanation or a modified sample code to the "Inverse problem for the Lorenz system" with added data loss is highly appreciated.

Thank you

**Comment by kyouma** [comment]
The "condition" on the data is set by sending an instance of the `dde.icbc.PointSetBC` class as another boundary/initial condition into dde.data.PDE together with `anchors=DATA_COORDINATES`. The loss function can be set in `model.compile(..., loss=..., ...)`. If you want a different loss function for the data, then you can do it like this: `loss=[loss, functions, for, other, components, DESIRED_LOSS]`, where the order of loss functions and the conditions in `dde.data.PDE` are the same. If you doesn't send a list to the `loss` parameter, then the same loss function for the DE, conditions and the data is used.
